hawkbit-runtime/hawkbit-update-server/src/main/resources/application-postgresql.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Enapter Co.,Ltd.
```

Comment:
Forgot to mention that you have to include a new licence header file as well in order to pass the licence validation, sorry for that. Just have a look [here](https://github.com/eclipse/hawkbit/tree/master/licenses) and add a new `LICENSE_HEADER_TEMPLATE_ENAPTER.txt` file accordingly